banzai/utils/date_utils.py

```diff
@@ -135,16 +135,14 @@ def valid_date(s):
 
 def get_min_and_max_dates(timezone, dayobs, return_string=False):
     midnight_at_site = _get_midnight(timezone, dayobs)
-    min_date = midnight_at_site - datetime.timedelta(days=0.5)
-    max_date = midnight_at_site + datetime.timedelta(days=0.5)
+    min_date = midnight_at_site + datetime.timedelta(days=0.5)
+    max_date = midnight_at_site + datetime.timedelta(days=1.5)
```

Comment:
I think this needs to be set to `min_date = midnight_at_site - timedelta(days=1)` and `max_date = midnight_at_site`. 

The reason this method was written to span midnight by +/- 0.5 days is because it has so far been used only in the e2e tests. The data for these tests is taken from when the same skyflat filter was used for evening / morning flats on the same night. Thus, for the tests you need to get flats from 1/2 a day before and after midnight. 

These days, you want to get the flats from the previous morning (so midnight minus one day) and then this evening (so up until this midnight.